examples/storage/s3/incremental_learning/README.md

```diff
@@ -1,10 +1,15 @@
 # Using Incremental Learning Job in Helmet Detection Scenario on S3
 
-This example based on the example: [Using Incremental Learning Job in Helmet Detection Scenario](/examples/incremental_learning/helmet_detection/README.md)
+This example based on the example: [Using Incremental Learning Job in Helmet Detection Scenario](/examples/incremental_learning/helmet_detection/README.md).    
 
-### Create a secret with your S3 user credential.
+### Prepare Nodes
+Assume you have created a [KubeEdge](https://github.com/kubeedge/kubeedge) cluster that have two cloud nodes(e.g., `Cloud_Node1`, `Cloud_Node2`)
+and one edge node(e.g., `Edge_Node`).  
+
+### Create a secret with your S3 user credential.   
 
 ```yaml

```

Comment:
If you want to add `kubectl`, then also update this code block language

Reply:
fix